components/FeatureFeed/FeatureFeed.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '../../ui-kit';
+
+import { FeatureProvider } from '../../providers';
+import {
+  HeroListFeature,
+  HorizontalCardListFeature,
+  VerticalCardListFeature,
+} from '../index';
+
+//This component is created to map the features by type and send them
+const FeatureFeed = (props = {}) => {
+
+  const CardListComponents = {
+    HeroListFeature,
+    HorizontalCardListFeature,
+    VerticalCardListFeature,
+  }
+
+  return props.data?.map((edge, i) => {
+    let { __typename } = edge;
+    if (!__typename && itemId) {
+      [__typename] = itemId.split(':');
+    }
```

Comment:
It might be nice to abstract this into a named function:

```js
function getComponent(edge) {
  let { __typename } = edge;

  if (!__typename && itemId) {
    [__typename] = itemId.split(':');
  }

  return CardListComponents[__typename];
}
```

Reply:
Great idea! I abstracted the function and added a `Components` variable so you can have it use different components. I put it in `/utils`